internal/pkg/util/fs/overlay/overlay.go

```diff
@@ -33,6 +33,7 @@ const (
 	fuse         = 0x65735546
 	ecrypt       = 0xF15F
 	lustre       = 0x0BD00BD0
+	gpfs         = 0x47504653
```

Comment:
This is the SUPER_MAGIC for GPFS. Handy list here: https://github.com/shirou/gopsutil/blob/master/disk/disk_linux.go